### DISCUSSION TITLE
Change APP_URL

### DISCUSSION THREAD

**Discussion by wefixit-AT** [snipe/snipe-it#10453] [Q&A] [answered]
Hi, I move my docker image to a new server and changed the hostname. Every think worked fine instead the images. The images are pointed to the old domain. I changed the APP_URL to reflect the new hostname. The problem persists on old and new images.

Does anybody know which steps have to be done when changing the hostname?

**Comment by adagioajanes** [comment]
Did you clear and recache your configuration with `php artisan config:cache`?

**Reply by wefixit-AT** [reply]
Sorry for the late response. This does not help.

**Comment by wefixit-AT** [comment, marked as answer]
Solution: I had to replace the url in /var/www/html/bootstrap/cache/config.php

**Reply by hainn15** [reply]
this fixed me! Thanks
